### PR TITLE
Fix error encountered while converting multi-language XLSForms to JSON

### LINKED ISSUE #513: Circular reference error raised when converting an XLSX form with multiple languages to JSON
#### Software and hardware versions 
pyxform v1.3.4

#### Problem description

The error `ValueError: Circular reference detected` is raised when trying to convert an XLSForm with multiple languages to JSON after running the `to_xml` function.

#### Steps to reproduce the problem

[XLSForm](https://docs.google.com/spreadsheets/d/1F1NKOLIZ_D-xCeZPEMez0OUBEHYEzWtoQjnC1ODIkpQ/edit?usp=sharing)

```
from pyxform.tests_v1.pyxform_test_case import PyxformTestCase

class TestConvertSurveyToJSON(PyxformTestCase):
    def test_able_to_convert_to_JSON(self):
        md = """
        | survey  |
        |         | type                   | name  | label:Eng  | label:Fr |
        |         | text                   | name  | Name       | Prénom   |
        |         | select_multiple fruits | fruit | Fruit      | Fruit    |
        |         |                        |       |            |          |
        | choices | list name              | name  | label:Eng  | label:Fr |
        |         | fruits                 | 1     | Mango      | Mangue   |
        |         | fruits                 | 2     | Orange     | Orange   |
        |         | fruits                 | 3     | Apple      | Pomme    |
        """

        survey = self.md_to_pyxform_survey(
            md_raw=md,
            kwargs={"id_string": "id", "name": "multi-language", "title": "some-title"},
            autoname=False,
        )
        survey.to_xml()
        survey.to_json()
```

#### Expected behavior

Forms with multiple languages should be converted to JSON correctly.

#### Other information 
Stack trace:
Screenshot 2021-01-26 at 12 42 29 PM

_Tried removing the `_translations` key and that seemed to fix the Circular import error; I believe the `parent` keys in the `_translations` dict may be raising the error._
Things you tried, stack traces, related issues, suggestions on how to fix it...

### PR DESCRIPTION
**Pull request by DavisRayM**
Closes #513

#### Why is this the best possible solution? Were any other approaches considered?

The simplest option removes objects that can't be converted to JSON.

#### What are the regression risks?

- N/A

#### Does this change require updates to documentation? If so, please file an issue [here](https://github.com/XLSForm/xlsform.github.io) and include the link below.

No

#### Before submitting this PR, please make sure you have:
- [x] included test cases for core behavior and edge cases in `tests_v1`
- [x] run `nosetests` and verified all tests pass
- [x] run `black pyxform` to format code
- [x] verified that any code or assets from external sources are properly credited in comments

### CODE REVIEW

File: pyxform/tests/xform2json_test.py [hunk 1]
```diff
@@ -50,6 +52,35 @@ def tearDown(self):
...
+            kwargs={"id_string": "id", "name": "multi-language", "title": "some-title"},
+            autoname=False,
+        )
+        expected_xml = survey.to_xml()
+        generated_json = survey.to_json()
+
+        survey = create_survey_element_from_dict(json.loads(generated_json))
+
+        self.assertEqual(expected_xml, survey.to_xml())
+
```
Comment (ukanga) [inline pyxform/tests/xform2json_test.py:83]: Can you move the test to `pyxform/tests_v1/ ` where all markdown based tests reside and should be? 

I expect at some point in future we will get rid of most tests in `pyxform/tests` and simply have one common `tests` module. As such, most if not all new tests should probably be in `pyxform/tests_v1/`.

**Review by ukanga — APPROVED** [review]
